static/js/browser2/observation_chart.tsx

```diff
@@ -96,6 +109,26 @@ export class ObservationChart extends React.Component<
       .on("mouseleave", this.handleDotLeave(svgContainerId));
   }
 
+  private fetchObservationDcidsData(): void {
+    let request = `/api/browser/observation-ids?place=${this.props.placeDcid}&statVar=${this.props.statVarId}`;
+    if (this.props.sourceSeries.measurementMethod) {
+      request =
+        request +
+        `&measurementMethod=${this.props.sourceSeries.measurementMethod}`;
```

Comment:
nit: request could be folded in too:

`${request}&measurementMethod...`

Reply:
done